test/powershell/Modules/Microsoft.PowerShell.Utility/WebCmdlets.Tests.ps1

```diff
@@ -2332,6 +2332,24 @@ Describe "Invoke-RestMethod tests" -Tags "Feature" {
         }
     }
 
+    Context "Invoke-RestMethod Single Value JSON null support" {
+        BeforeAll {
+            $baseUrl = 'http://localhost:8081/PowerShell?test=response&contenttype=application/json&output='
+        }
+        It "Invoke-RestMethod Supports a Single Value JSON null" {
+            $url = '{0}{1}' -f $baseUrl, 'null'
+            Invoke-RestMethod -Uri $url | Should Be $null
+        }
+        It "Invoke-RestMethod Supports a Single Value JSON null and ignores whitespace" {
+            $url = '{0}{1}' -f $baseUrl, "            null         "
+            Invoke-RestMethod -Uri $url | Should Be $null
+            $url = '{0}{1}' -f $baseUrl, "            
+                                            null         
```

Comment:
I would prefer to use `` `n`` for newline whitespace than rely on the line endings of the file

Reply:
fixed.